angr/analyses/loopfinder.py

```diff
@@ -25,7 +25,7 @@ def __init__(self, entry, entry_edges, break_edges, continue_edges, body_nodes,
                     break
 
     def __repr__(self):
-        s = "<Loop @ %s, %d blocks>" % (self.entry.addr, len(self.body_nodes))
+        s = "<Loop @ %#x, %d blocks>" % (self.entry.addr, len(self.body_nodes))
```

Comment:
`%#x` will fail if `self.entry.addr` is a string, which might the case because of Java support (gah).